azext_edge/edge/_help.py

```diff
@@ -807,40 +752,39 @@ def load_iotops_help():
         examples:
         - name: Show the details of an asset.
           text: >
-            az iot ops asset show --name {asset_name} -g {resource_group}
+            az iot ops asset show --name myasset -g myresourcegroup
     """
 
     helps[
         "iot ops asset update"
     ] = """
         type: command
         short-summary: Update an asset.
-        long-summary: To update data points and events, please use the command groups `az iot ops asset data-point` and
-            `az iot ops asset events` respectively.
+        long-summary: To update datasets and events, please use the command groups `az iot ops asset dataset` and
+            `az iot ops asset event` respectively.
 
         examples:
-        - name: Update an asset's data point and event defaults.
+        - name: Update an asset's dataset and event defaults.
           text: >
-            az iot ops asset update --name {asset_name} -g {resource_group} --data-publish-int {data_point_publishing_interval}
-            --data-queue-size {data_point_queue_size} --data-sample-int {data_point_sampling_interval}
-            --event-publish-int {event_publishing_interval} --event-queue-size {event_queue_size}
-            --event-sample-int {event_sampling_interval}
+            az iot ops asset update --name myasset -g myresourcegroup --dataset-publish-int 1250 --dataset-queue-size 2 --dataset-sample-int 30
+            --event-publish-int 750 --event-queue-size 3 --event-sample-int 50
 
-        - name: Update an asset's asset type, description, documentation uri, external asset id, hardware revision, product code,
+        - name: Update an asset's description, documentation uri, hardware revision, product code,
                 and software revision.
           text: >
-            az iot ops asset update --name {asset_name} -g {resource_group} --asset-type {asset_type} --description {description}
-            --documentation-uri {documentation_uri} --external-asset-id {external_asset_id} --hardware-revision {hardware_revision}
-            --product-code {product_code} --software-revision {software_revision}
+            az iot ops asset update --name myasset -g myresourcegroup --description "Updated test asset description."
+            --documentation-uri www.help2.com --hardware-revision 11.0
```

Comment:
nit: I feel like we should use a known URL (bing.com, azure.com, contoso.com/help, etc) for the example documentation URI just in case someone decides to purchase that one :)

Reply:
sure